### ISSUE TITLE
size settings

### ISSUE THREAD

**Issue by Gui-HBS** [ModernFlyouts-Community/ModernFlyouts#1279]
### Description of the new feature / enhancement

make it possible to configure the modernFlyouts display size, I found the size a little big and I didn't find anything about decreasing it :(

### Scenario when this would be used?

on full screens, for example

**Comment by Synfoxic** [comment]
YES! I just found out about ModernFlyouts and it is perfect, except that the size is a little too big for me too. I am looking for a sleek and small controller.

**Comment by Gui-HBS** [comment]
> YES! I just found out about ModernFlyouts and it is perfect, except that the size is a little too big for me too. I am looking for a sleek and small controller.

Hi! Are you yet using ModernFlyouts? it wasn't working so well in Windows 11, so im using the RainMeter instead. Looks like the ModernFlyouts is no longer receiving updates, which is very sad, because i think ModernFlyouts is an awesome software and way better than RainMeter.
